### DISCUSSION TITLE
Jeedoo Scraper Error

### DISCUSSION THREAD

**Discussion by GMendes73** [adultmm/AdultMediaManager#80] [General]
Hello. Thanks for the incredible work!

I have this issue: some movies, sometime part of a series, causes an error:  

Scraper: Jeedoo 
The connection is closed in an unexpected way

After that message, another one: Undefined object reference for an object instance

(The original message is in portuguese, my default language)

![Captura de tela 2023-01-31 041608](https://user-images.githubusercontent.com/124029577/215692568-93c411ad-7342-468c-9843-b289da6f22a7.png)

**Comment by adultmm** [comment]
Hi @GMendes73, 

This error has been fixed in [v0.12.0-BETA.2](https://github.com/adultmm/AdultMediaManager/releases/tag/v0.12.0-BETA.2). 

Enjoy!
